README.md

```diff
@@ -203,6 +203,11 @@ options:
   request_review:
     # False by default
     enabled: true
+    # If admins are required for review, they can be automatically selected.
+    # Admin of a repo is granted in three ways: direct user admin on the Repository, team admin on the repository,
+    # or as an Organization Owner.
+    # To reflect these permission sources, admin_scope can be set to (user|team|org)
+    admin_scope: "org"
```

Comment:
Are you sure you want to commit to this configurability? Seems fine to keep the settings internally, but I'd be tempted to just hard-code this setting so that only people who have admin perms via the `user` or `team` scope get requested (and org level admins never get requested).

Reply:
The more I think about it, the more I think you're right. We don't need to expose the complexity today, and we can always make it more configurable later. I'll push up a commit to reflect this.